docs/tasks/administer-cluster/kubeadm-upgrade-1-8.md

```diff
@@ -202,6 +202,12 @@ $ kubeadm upgrade apply v1.8.0
    find your CNI provider and see if there are additional upgrade steps
    necessary.
 
+6. Add RBAC permissions for automated certificate rotation. In the future, kubeadm will perform this step automatically:
+
+```shell
+$ kubectl create clusterrolebinding autoapprove-node-client-rotation --clusterrole=system:certificates.k8s.io:certificatesigningrequests:selfnodeclient --group=system:nodes
```

Comment:
let's use name "kubeadm:node-autoapprove-certificate-rotation" ? so, once we will have fix in kubeadm, it will be same names for bindings, created either manually or automatically.

Reply:
SGTM, will change.